docs/main_text.md

```diff
@@ -156,14 +156,14 @@ The refine.bio processed Salmon indices are available for download.
 You can make use of our API like so:
 
 ```
-https://api.staging.refine.bio/transcriptome_indices/?organism=<ORGANISM>&length=<LENGTH>
+https://api.refine.bio/transcriptome_indices/?organism=<ORGANISM>&length=<LENGTH>
 ```
 Where `ORGANISM` is the scientific name of the species in all caps separated by underscores and `LENGTH` is either `SHORT` or `LONG`.
```

Comment:
Do you think it will be better to use `<ORGANISM>` and `<LENGTH>` on this line? I am afraid a user may type the URL like this:
```https://api.refine.bio/transcriptome_indices/?organism=<HOMO_SAPIENS>&length=<LONG>```